libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts

```diff
@@ -37,4 +37,8 @@ export class SuggestionsListItemComponent implements FocusableOption, OnInit {
     unfocus() {
         this.element.nativeElement.style = '';
     }
+
+    scrollIntoView() {
+        this.element.nativeElement.scrollIntoViewIfNeeded(false);
```

Comment:
We need to support FF: https://caniuse.com/?search=scrollIntoViewIfNeeded

Reply:
I changed `scrollIntoViewIfNeeded` for `scrollIntoView` which is supported in FF (https://caniuse.com/?search=scrollIntoView)